.travis.yml

```diff
@@ -9,10 +9,12 @@ cache:
 
 env:
   global:
-    - ANSIBLE_HOST_KEY_CHECKING=False
-    - PIP_DOWNLOAD_CACHE=$HOME/.cache/pip
-    - OC_BINARY_URL=https://mirror.openshift.com/pub/openshift-v3/clients/3.10.45/linux/oc.tar.gz
+    - ANSIBLE_HOST_KEY_CHECKING="False"
+    - PIP_DOWNLOAD_CACHE="$HOME/.cache/pip"
+    - OC_BINARY_URL="https://mirror.openshift.com/pub/openshift-v3/clients/3.11.133/linux/oc.tar.gz"
```

Comment:
Good change. I know there was issues with OC client v3.11.x for earlier CI runs and it has been holding us back from moving to v3.11, but hopefully those issues have been resolved now (wonder if it was an issue with Travis CI env actually and hence we should be good when moving away).